katsdptelstate/aio/redis.py

```diff
@@ -170,6 +170,16 @@ def __init__(self, client: aioredis.Redis) -> None:
                 'katsdptelstate', 'lua_scripts/{}.lua'.format(script_name))
             self._scripts[script_name] = _Script(script)
 
+    @classmethod
+    async def from_url(cls, url: str) -> 'RedisBackend':
+        """Create a backend from a redis URL.
+
+        This is the recommended approach as it ensures that the server is
+        reachable, and sets some timeouts to reasonable values.
+        """
+        client = await aioredis.create_redis_pool(url, timeout=5)
```

Comment:
You mention "health check interval" in the commit message but I don't see it here. Is it using a default value instead of 30 seconds?

Reply:
That'll be because I cherry-picked the commit from the aioredis2 branch and edited it. aioredis 1.x didn't have health check intervals as far as I know.